services/pypi/pypi-python-versions.tester.js

```diff
@@ -3,7 +3,6 @@
 const Joi = require('@hapi/joi')
 const t = (module.exports = require('../tester').createServiceTester())
 
-// These regexes are the same, but declared separately for clarity.
```

Comment:
This probably referred to `isPipeSeparatedDjangoVersions` but these are no longer in the same file.